packages/ciphernode/enclave/src/password_set.rs

```diff
@@ -40,15 +40,15 @@ fn get_zeroizing_pw_vec(input: Option<String>) -> Result<Zeroizing<Vec<u8>>> {
     Ok(pw)
 }
 
-pub async fn execute(config: &AppConfig, input: Option<String>, overwrite: bool) -> Result<()> {
+pub async fn execute(config: &AppConfig, input: Option<String>) -> Result<()> {
     println!("Setting password...");
-    enclave_core::password::create::preflight(config, overwrite).await?;
+    enclave_core::password::create::preflight(config).await?;
```

Comment:
`nit`: maybe we could rename this file to set as well for consistency?

Reply:
you mean `password_set.rs`? I think it should be called like this since we could have other commands that can have the `set` subcommand that may overlap!

Reply:
no no i mean, `password::create` should be `password::set`